scripts/orchestration/creative_pilot_workspace.py

```
@@ -29,27 +31,62 @@
     prepare as prepare_specification,
+    prepare_exact as prepare_exact_specification,
+    validate_default_prepare_artifact_snapshots,
+    validate_exact_prepare_artifact_snapshots,
+    validate_exact_prepare_artifacts,
 )
 from scripts.orchestration.creative_hypothesis_spec_bridge_contract import (
     CreativeHypothesisSpecBridgeError,
     build_creative_pilot_spec_bridge_bundle,
 )
 from scripts.orchestration.creative_code_contract import CreativeCodeContractError
+from scripts.orchestration.creative_code_specification import (
```

Flake8 (7.3.0)

[error] 'scripts.orchestration.creative_code_specification.validate_source_candidate_packet' imported but unused

(F401)